From 0c76611aa48b3942a9c45048ab6fd5e68e63bbcd Mon Sep 17 00:00:00 2001
From: Steve deRosier <steve.derosier@lairdtech.com>
Date: Thu, 3 Mar 2016 17:23:48 -0800
Subject: [PATCH] laird: dts: wb50n: Add gpio btn0 for GPIO-2

This adds a gpio_keys button to GPIO-2 (BB40 designation) so that we can
wake from suspend.

Bug: 9116
---
 arch/arm/boot/dts/wb50n.dts | 10 ++++++++--
 1 file changed, 8 insertions(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/wb50n.dts b/arch/arm/boot/dts/wb50n.dts
index ecbc742..8a39a8b 100644
--- a/arch/arm/boot/dts/wb50n.dts
+++ b/arch/arm/boot/dts/wb50n.dts
@@ -235,12 +235,18 @@
 		compatible = "gpio-keys";
 		#address-cells = <1>;
 		#size-cells = <0>;
+		btn0@pa10 {
+			label = "BTNESC";
+			linux,code = <1>; /* ESC button */
+			gpios = <&pioA 10 GPIO_ACTIVE_LOW>;
+			gpio-key,wakeup = <1>;
+		};
+
 		irqbtn@pe31 {
 			label = "IRQBTN";
-			linux,code = <99>;
+			linux,code = <99>; /* SysReq button */
 			gpios = <&pioE 31 GPIO_ACTIVE_LOW>;
 			gpio-key,wakeup = <1>;
 		};
 	};
-
 };
-- 
1.9.1

